docs/api/source/_autoapi_templates/python/attribute.rst

```diff
@@ -0,0 +1 @@
+{% extends "python/data.rst" %}
```

Comment:
These and other .rst files control the layout of the documentation and are copy-and-pasted from defaults, for future adjustments if necessary.